_posts/2015-07-03-joan-introduction.md

```diff
@@ -0,0 +1,16 @@
+---
+layout: post
+title: "Introduction: Joan"
+permalink: /2015-07-03-joan-introduction
+author: Joan Wang
+---
+
+My name is Joan, and I am one of four student interns working on identifying the barriers and facilitators for homeless families to secure permanent housing, in collaboration with the Gates Foundation. Here’s a bit about myself: I am currently pursuing an EU-funded dual master’s degree in spatial planning and environmental policy at Cardiff University (Wales, UK). Before that, I completed an interdisciplinary bachelor’s degree in mathematics-economics from Reed College (Portland, OR). I have a social-sciences background with a predominantly quantitative and data-intensive skillset, and I have interests in urban economics and sustainability. In particular, my past research projects and work experiences have revolved around energy efficiency and urban food systems.
+
+Having grown up in Bellevue just across Lake Washington, I always keep an eye out for innovative projects that are sprouting up in the Greater Seattle area—especially those that connect with surrounding urban communities—so naturally I jumped at the chance to participate in the Data Science for Social Good (DSSG) program. When I was writing the motivation letter, I had to refrain from frantically typing: Data science! Urban studies! Working on real-world projects that combine these two is a life-long dream!  I believe what I ended up with was this vague idea that DSSG allows me to use “mixed data analytics and critical social science … to help understand – and solve – complex real-world issues facing Puget Sound communities today.”
+
+Three weeks into the program, I have learned a lot about myself and a variety of topics. I believe this is a result of DSSG’s impressive support system for the student interns, be it technical, professional, social, or even nutritional! Working with Neil Roche and Anjana Sundaram, our project leads from the Gates Foundation, has been an invaluable experience where despite having to confront the assumptions and limitations of both philanthropic research and our own professional fields, we still endeavor towards a common language and understanding. Moreover, on top of lending us their technical expertise, the two data scientists on our team—Ariel Rokem and Bryna Hazelton—have fostered a working atmosphere that is rooted in critical thinking and effective communication, which has been incredibly motivating. 
+
+I haven’t even talked about the awesome colleagues I work with everyday. It will have to be another blogpost to do it justice. Suffice to say that having at least four people leading me step-by-step, I was able to execute commands in a completely new programming language to make several non-trivial discoveries about our data.
```

Comment:
Fortunately, they are writing their own blog-posts to introduce themselves :-) 

Further thoughts about team-work in this context would be very welcome. The premise of DSSG is that bringing together people with different backgrounds has a synergistic effect. But do we ultimately spend more time bridging across disciplinary and professional boundaries? Is the format the right one to capitalize on the potential for synergy?